src/token.c

```diff
@@ -9,6 +9,9 @@
 #include <stdlib.h>
 #include <stdbool.h>
 #include <unistd.h>
+#ifdef UCLIBC
```

Comment:
you should be using `__UCLIBC__` here  thats the correct compiler defined macro.